### ISSUE TITLE
Provide a guide on how to make themes

### ISSUE THREAD

**Issue by puelp** [krateng/maloja#286]

**Comment by duckfromdiscord** [comment]
It seems to just be a bunch of CSS things that you change the hex colors for. Reading the CSS here: https://github.com/krateng/maloja/blob/master/maloja/web/static/css/themes/constantinople.css or here: https://github.com/krateng/maloja/blob/master/maloja/web/static/css/themes/kda.css shows how to do it, but there is some extra fadeouts and stuff that aren't as intuitive which would require you to understand CSS which is a whole different thing.

You'd just copy one of those files and make a new one of a different name in the same folder, with some modifications.

**Comment by puelp** [comment]
Yeah I guess if you have some CSS knowledge, it may seem simple, but for casual users, it's hard because you have no idea which field changes what, and then you have to copy and paste each value into a color number tool to see what color it is, and so on.

Maybe if there were comments like "# This is used as the background color on all pages." or "# This changes the text color of the track titles." on each line, I feel like that would be helpful.

**Comment by duckfromdiscord** [comment]
A template with comments like that could work! I wish I could make some sort of theme maker that could do this interactively/live with a GUI but that would require interfacing with the maloja UI and changing it externally somehow. Or I could just make something that has a list of colors with color pickers that exports to CSS.
